nym-wallet/src/pages/bonding/node-settings/settings-pages/general-settings/InfoSettings.tsx

```diff
@@ -0,0 +1,245 @@
+import { useEffect, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { Button, Divider, Typography, TextField, Grid, Alert, IconButton, CircularProgress } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import CloseIcon from '@mui/icons-material/Close';
+import { updateMixnodeConfig } from '../../../../../requests';
+import { TBondedMixnode, TBondedGateway } from '../../../../../context/bonding';
+import { SimpleModal } from '../../../../../components/Modals/SimpleModal';
+import { bondedInfoParametersValidationSchema } from '../../../../../components/Bonding/forms/mixnodeValidationSchema';
+
+export const InfoSettings = ({ bondedNode }: { bondedNode: TBondedMixnode | TBondedGateway }) => {
+  const [open, setOpen] = useState(true);
+  const [openConfirmationModal, setOpenConfirmationModal] = useState<boolean>(false);
+
+  const theme = useTheme();
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors, isSubmitting, isDirty, isValid },
+  } = useForm({
+    resolver: yupResolver(bondedInfoParametersValidationSchema),
+    mode: 'onChange',
```

Comment:
I'm not sure if you need this. See if it works without.

Reply:
@fmtabbara I need it because if the values don't change the button should be disable, if not, and the user send the same parameters already set-up, and is received an error from the api